pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitScanErrorTest.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.dsm.kits;
+
+import org.broadinstitute.dsm.model.kit.ScanError;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class KitScanErrorTest {
+
+    @Test
```

Comment:
Can you please add a test for error = null and shortId =null?

Reply:
One of those tests actually brought up a weak point and I added an extra check in the method for `shortId = null` case, thank you!!